documents/coredocument.go

```diff
@@ -263,7 +263,7 @@ func (cd *CoreDocument) Patch(documentPrefix []byte, collaborators Collaborators
 	wcs := collaborators.ReadWriteCollaborators
 	rcs = append(rcs, wcs...)
 
-	ncd := &CoreDocument{Document: cdp}
+	ncd := &CoreDocument{Document: cdp, Status: Pending}
```

Comment:
Since we are already setting this value in the `documents` package in other places, lets keep it like this, but lets review this approach soon.

Reply:
this is a hack until we move the pending status from documents since patch is not setting status